tests/test_cameraFilter.py

```diff
@@ -1,8 +1,15 @@
-
-from vgc.cameraFilter import cameraFilter 
+from time import sleep
```

Comment:
The test file should also follow the PEP-8 standard. Make sure you add docstrings etc

Reply:
The tests for the camera filter are not done and therefore not pushed.

Reply:
The test file shouldn't have been part of this pullrequest at all